doc/api/events.md

```diff
@@ -586,6 +586,53 @@ to indicate an unlimited number of listeners.
 
 Returns a reference to the `EventEmitter`, so that calls can be chained.
 
+### async emitter.when(eventName[, options])
```

Comment:
If it's just returning a Promise, do we really need to add the 'async ' prefix here? None of the examples even utilize async/await.

Reply:
I was going back and forth on this. It depends on what (if any) convention we want to set on this. I'm not completely set on it and I expected that if anyone didn't like it then they'd speak up :-)

Reply:
async/await works for any `Promise` right? If so, I'm not sure we need to explicitly add the prefix in that case.

Reply:
I do like the `async` prefix.

Reply:
I like the async prefix too but think that it's important we're consistent with the other method returning promises in the API (util.promisify)